src/Pagination/PaginationButton.tsx

```diff
@@ -54,6 +54,7 @@ class PaginationButton extends React.Component<PaginationButtonProps> {
     const item = (
       <Component
         {...unhandled}
+        active={active}
```

Comment:
The default value of the componentClass property is SafeAnchor. SafeAnchor has no active property, which will cause a warning.
![image](https://user-images.githubusercontent.com/1203827/73707421-7ed64880-4736-11ea-8b81-427c3a08f08f.png)